### PR TITLE
Fix import/export

### PR DESCRIPTION
**Pull request by andersonluisribeiro**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix import data preserving the id of the records

* **What is the current behavior?** (You can also link to an open issue here)
Import data doesn't work properly

* **What is the new behavior (if this is a feature change)?**
Import data preserving the id of records

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This is connected to dojot/dojot#922

* **Other information**:

### CODE REVIEW

File: src/services/requestsImport.js [hunk 1]
```diff
@@ -1,123 +1,13 @@
 import { logger } from '@dojot/dojot-module-logger';
-import requestsDevice from './requestsDevice';
-import requestsTemplate from './requestsTemplate';
 import requestFlow from './requestsFlow';
 import Requests from '../utils/requests';
 import config from '../config';
 
 async function deleteAllData(token) {
```
Comment (eliasreis54) [inline src/services/requestsImport.js]: I think the name can be deleteAllFlows... What do you think?

**Comment by eliasreis54** [comment]
I would like to know... Why templates and devices are not be imported any more?

File: src/services/requestsImport.js (hunk 1 above)
Comment (andersonluisribeiro) [reply src/services/requestsImport.js]: Hi Elias, how are you? From now on we are importing the devices and templates into the same json. We only sent one request to the device-manager. We are also considering ids on import.

**Comment by giovannicuriel** [comment]
Hi :sunglasses: !
This feature is moving to DeviceManager. There, we could do things like.... :tada: preserving IDs :tada:

In fact, we are planning to add to all services that somehow deal with persisted data (such as flowbroker, history, etc.) import-export endpoints. Therefore, data-manager could be an orchestrator for importing and exporting data from dojot as a whole.

File: src/services/requestsImport.js (hunk 1 above)
Comment (andersonluisribeiro) [reply src/services/requestsImport.js]: You're right, deleteAllFlows is more cohesive, I'll make the change

**Pull request merged**